src/azure-cli/azure/cli/command_modules/config/tests/latest/test_config.py

```diff
@@ -13,6 +13,7 @@
 
 class ConfigTest(ScenarioTest):
 
+    @unittest.skip('skip')
```

Comment:
Maybe you can replace 'skip' with a real reason.

Reply:
It's already outdated.